src/pages/blog/index.astro

```diff
@@ -0,0 +1,26 @@
+---
+import Main from "../../layouts/Main.astro";
+import CardSection from "../../components/CardSection.vue";
+import Card from "../../components/Card.vue";
+import Blogroll from "../../components/Blogroll.vue";
+
+let posts = await Astro.glob("./*.md");
+posts = posts.sort((a, b) => {
+	return +new Date(b.frontmatter.pubDate) - +new Date(a.frontmatter.pubDate);
+});
+---
+
+<Main title="Blog" description="Jonathan Bell's thoughts and feelings">
+	<CardSection section-title="Blog">
+		{posts.map((post) => (
```

Comment:
how very react. Or is astro just JSX?

Reply:
I think so, yes. Astro uses JSX but we can use Svelte or Vue if we want. For the routing (`index.astro`) I decided to keep it simple and just embrace the framework ; )